src/pages/HomePage.vue

```diff
@@ -2,46 +2,53 @@
   <q-page class="flex column justify-between home-page">
     <div class="fit flex justify-center items-center home-banner">
       <q-img
-          src="icons/logo_modelizer.svg"
-          class="home-logo"
-          :ratio="1"
+        src="icons/logo_modelizer.svg"
+        class="home-logo"
+        :ratio="1"
       />
     </div>
-    <div class="flex home-content">
+    <div class="column items-center home-content">
       <q-btn
-          color="primary"
-          size="xl"
-          icon="fa-solid fa-circle-plus"
-          data-cy="createEmptyProject"
-          :label="$t('actions.home.createEmptyProject')"
-          :to="`/modelizer/${project.id}/model`"
+        color="primary"
+        size="xl"
+        icon="fa-solid fa-circle-plus"
+        data-cy="create-empty-project"
+        :label="$t('actions.home.createEmptyProject')"
+        :to="`/modelizer/${project.id}/model`"
+        @click="updateProject(project)"
```

Comment:
Here, you don't update any project cause it's not created yet in the localstorage. Here you should use the `createProject(project)`

Reply:
But `createProject` and `updateProject` would have the same body